### PR TITLE
Remove Pydantic as dependency

### PR DESCRIPTION
**Pull request by pinkwah**
Resolves: #475

### CODE REVIEW

**Comment by TeeeJay** [comment]
Sorry to bother, but would it make sense to add <3 or ^2 as a dependency resolver instead? That is what we in eCalc ...

**Comment by pinkwah** [comment]
I am not 100% sure about this, but I think `^2` is a Poetry thing and not Python. You are correct that it would be better to have an upper limit, but maintaining this without Poetry would be too difficult.

File: setup.py [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "autoapi",
     "ipython",
     "rstcheck",
-    "pydantic<2",  # remove this when ert unpins fastapi and pydantic
+    "pydantic",
```
Comment (berland) [inline setup.py:34]: Remove the entire line, as pydantic is a transitive dependency only.

**Review by berland — APPROVED** [review]
👍🏻

**Pull request merged**